components/WhatICanHelpYouWith/organisms/WhatICanHelpYouWith.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  Green050, Grey010,
+} from 'components/styling/colors';
+
+import PrimaryCTAButton from 'components/atoms/PrimaryCTAButton';
+import { whatICanHelpYouWithItems } from 'components/WhatICanHelpYouWith/content/whatICanHelpYouWithItems';
+import WhatICanHelpYouWithUnit from 'components/WhatICanHelpYouWith/molecules/WhatICanHelpYouWithUnit';
+import WhatICanHelpYouWithTop from '../molecules/WhatICanHelpYouWithTop';
+
+import {
+  Container,
+  LowerContainer,
+  MobileCTAContainer,
+} from './WhatICanHelpYouWith.styles';
+
+const WhatICanHelpYouWith = () => (
+  <Container>
+    <WhatICanHelpYouWithTop />
+    <LowerContainer>
+      {whatICanHelpYouWithItems.map(
+        (item, index) => <WhatICanHelpYouWithUnit item={item} index={index} />,
```

Comment:
I think it is better to call the index `key` to make it trivial for React devs whacca doing here

Reply:
Oh yeah it's a simple mistake I missed